gtm/README.md

```diff
@@ -0,0 +1,141 @@
+# Search Insights for Google Tag Manager
+
+Google Tag Manager [custom template](https://developers.google.com/tag-manager/templates/) for Algolia Search Insights.
+
+---
+
+<div align="center">
+
+[**Download template**](generated/search-insights.tpl)
+
+</div>
+
+---
+
+## Usage
+
+### 1. Import the template
+
+- Download **[`search-insights.tpl`](generated/search-insights.tpl)**
+- Import the file in [Google Tag Manager](https://tagmanager.google.com) (GTM)
+  - Go to "Templates"
+  - Click on "New"
+  - Select "Import" in the menu
+
+### 2. Create variables
+
+For events to be sent with the correct information (`userToken`, `objectIDs`, `queryID`, etc.), you need to add variables in the "User-Defined Variables" section.
+
+[Learn more about GTM variables →](https://www.simoahava.com/analytics/variable-guide-google-tag-manager/)
```

Comment:
is there an official guide on this?

Reply:
Simo Ahava is the reference for this for now, he basically writes the documentation that Google is missing...